src/v/model/namespace.h

```diff
@@ -75,4 +75,14 @@ inline const model::topic_partition schema_registry_internal_tp{
 inline const model::ntp wasm_binaries_internal_ntp(
   model::redpanda_ns, model::topic("wasm_binaries"), model::partition_id(0));
 
+inline bool is_user_topic(const topic_namespace_view& tp_ns) {
```

Comment:
the const ref is not required, it is already a view

Reply:
fixed